### PR TITLE
fixed prevVersion detection on windows

### PR DESCRIPTION
**Pull request by staabm**

### CODE REVIEW

**Comment by staabm** [comment]
after applying this fixed, I also found that contributors.txt is sprinkeled with single quotes. I guess we needs this kind of fix in several spots:

```
'Brahim Arkni
'Christopher Bauer
'Daniel Orner
'Denisson Padilha
'dorner
'Ehsan Abidi Ashtiani
'Filip Mares
'Jörn Zaefferer
'Julio Spader
'KJ
'Lars Skjelbek
'Lukasz Wozniak
'Markus Staab
'Martti Laine
'Michal Moravec
'netzkind
'rasstislav
'sadj
'Salahuddin Hairai
'Saldanha
'silovski
'The Gitter Badger
'Thomas Gohard
'unknown
'Wojciech Wałek
'Štefan Baebler
Alok Saldanha'
Brahim Arkni'
Christopher Bauer'
Daniel Orner'
Jörn Zaefferer'
Julio Spader'
Markus Staab'
Martti Laine'
sadj'
Salahuddin Hairai'
Thomas Gohard'
```

**Comment by staabm** [comment]
adjusted the PR with some more fixes. As is I can build jquery-validate on my windows machine

**Comment by staabm** [comment]
see http://stackoverflow.com/questions/24173825/what-does-single-quote-do-in-windows-batch-files/24181667#24181667 for background

File: lib/repo.js [hunk 1]
```diff
@@ -237,14 +237,14 @@ Release.define({
 		Release.chdir( Release.dir.repo );
 		console.log( "Committing release artifacts..." );
 		Release.exec( "git add -f " + paths.join( " " ), "Error adding release artifacts to git." );
-		Release.exec( "git commit -m '" + Release.newVersion + "'",
+		Release.exec( "git commit -m \"Release:" + Release.newVersion + "\"",
```
Comment (Arkni) [inline lib/repo.js]: no need to prefix the commit with `Release:`

Comment (staabm) [inline lib/repo.js]: I will check that again. I guess I used somehow a old version of commitplease (maybe because of outdated nodejs or npm)

Comment (staabm) [inline lib/repo.js]: verified in #88, this is no longer required as the version is (using this PR) properly dected (without the single quotes)

**Comment by staabm** [comment]
just tried to verify mit CLA, but the link posted by the BOT above leads to a 404 page.
(I already signed the CLA for my contributions for jquery.js in the past)

**Comment by jzaefferer** [comment]
Re CLA: You've signed with markus.staab@redaxo.de, but your git email is maggus.staab@googlemail.com. You could sign again with the other email or update your commit.

**Comment by staabm** [comment]
here we go, @jzaefferer thx for the heads up

**Comment by jzaefferer** [comment]
Changes look fine to me. Would be useful to update the commit message to explain why this is necessary (link to Stackoverflow?).

**Comment by staabm** [comment]
agree. adjusted per feedback.

**Comment by jzaefferer** [comment]
Thanks!

**Pull request closed without merging**